.github/workflows/deploy-docs.yml

```diff
@@ -41,6 +41,10 @@ jobs:
           yarn example:build
           yarn example-form:build
           yarn docs:build
+        env:
+          # 7 GiB by default on GitHub, setting to 6 GiB
+          # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
+          NODE_OPTIONS: --max-old-space-size=6144
```

Comment:
We need to bump the memory in our GitHub actions when building storybook. If left to it's default, storybook fails to build with a "out of memory" error.